manifests/s/StrawberryPerl/StrawberryPerl/5.38.0.1/StrawberryPerl.StrawberryPerl.yaml

```
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.2.8 $debug=NVS0.CRLF.7-3-6.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: StrawberryPerl.StrawberryPerl
+PackageVersion: 5.38.0.1
```

```suggestion
PackageVersion: 5.38.1
```
This must match Control Panel.